src/sagas/abandon.saga.ts

```diff
@@ -0,0 +1,50 @@
+import { Message, MessageHandler, mediator } from 'mediateur';
+import { Event } from '@potentiel-infrastructure/pg-event-sourcing';
+import { Abandon } from '@potentiel-domain/laureat';
+import { publishToEventBus } from '../config/eventBus.config';
+import { ProjectAbandoned } from 'src/modules/project';
```

Comment:
import { ProjectAbandoned } from '../modules/project';
```suggestion
import { ProjectAbandoned } from 'src/modules/project';
```